test/helpers/deployment.js

```diff
@@ -31,11 +42,15 @@ export const getAwsBucketClient = (region = getAwsRegion()) => new S3Client({
 export const getApiEndpoint = () => {
   // CI/CD deployment
   if (process.env.SEED_APP_NAME) {
+    const stage = getStage()
     return `https://${stage}.up.web3.storage`
   }
 
   const require = createRequire(import.meta.url)
-  const testEnv = require('../../.test-env.json')
+  const testEnv = require(path.join(
+    process.cwd(),
+    '.sst/outputs.json'
```

Comment:
this was changed from new version, but of course too internal to be included in the migration guide... one of those "needle in a haystack" that took forever to get tests working with dynamic deployment of infra per PR stage